polyfill/spatial-navigation-polyfill.js

```diff
@@ -39,42 +39,33 @@
      * CSS.registerProperty() from the Properties and Values API
      * Reference: https://drafts.css-houdini.org/css-properties-values-api/#the-registerproperty-function
      */
-    if (window.CSS && CSS.registerProperty &&
-      window.getComputedStyle(document.documentElement).getPropertyValue('--spatial-navigation-contain') === '') {
-      CSS.registerProperty({
-        name: '--spatial-navigation-contain',
-        syntax: 'auto | contain | delegable',
-        inherits: false,
-        initialValue: 'auto'
-      });
-    }
+    if (window.CSS && CSS.registerProperty) {
```

Comment:
`window.CSS && CSS.registerProperty` may also be unnecessary similar  to the case of '--spatial-navigation-action' and '--spatial-navigation-function'.

Reply:
Oh, this was my mistake. No problem on this!